server/src/envelope.rs

```diff
@@ -178,6 +178,16 @@ pub enum AttachmentType {
     #[serde(rename = "event.applecrashreport")]
     #[allow(dead_code)]
     AppleCrashReport,
+
+    #[serde(rename = "event.event")]
+    /// A msgpack-encoded event submitted as part of minidump uploads.
+    Event,
+
+    #[serde(rename = "event.breadcrumb")]
+    /// This is a special attachment that can contain breadcrumbs encoded as message pack. There can be
+    /// two attachments that the SDK may use as swappable buffers. Both attachments will be merged and
+    /// truncated to the maxmimum number of allowed attachments.
+    Breadcrumb,
```

Comment:
_nit_: Plural (also in the serde rename)

Reply:
I also renamed `AttachmentType::Event` to something else to prevent confusion with `ItemType::Event`

Reply:
yeah I also renamed `Event` now